packages/api/src/resolvers/article/index.ts

```diff
@@ -257,6 +258,9 @@ export const createArticleResolver = authorized<
 
       const saveTime = new Date()
       const slug = generateSlug(parsedContent?.title || croppedPathname)
+      const wordsCount = wordsCounter(parsedContent?.content || '', {
+        isHtml: true,
+      }).wordsCount
```

Comment:
nice. I'm picturing in the future we could have a bunch of metadata on pages that we expose through a REST API, like get /<pageid>/metadata and it just returns key value data for things like word count.

Reply:
I guess one worry is does this function throw on error, or just return zero? We probably don't want to completely fail if an error occurs.

Reply:
I checked the source code and what it does is convert HTML to text and count words.
It is possible that the converting throws an error so I will write an exception handler to capture any error and return 0 instead.

Yeah, totally agree with the REST API for metadata!